api/setup.cfg

```diff
@@ -1,7 +1,2 @@
-[pycodestyle]
-max_line_length=120
-exclude=migrations
-
-[isort]
```

Comment:
Does black deal with import order? Or is it handled by flake8?

Reply:
No, actually another commit hook takes care of that. If you look at the [pre-commit config file](https://github.com/openoakland/woeip/pull/143/files#diff-82c3b36709926de3a1b27bea0d8258cc) you'll see the reorder-sort repo linkage. You can read more about that [here](https://github.com/asottile/reorder_python_imports)

Reply:
Ah I see now. Haven't used that one either, but I'll trust you on using it over `isort`.